workflow/executor/docker/docker.go

```diff
@@ -86,6 +89,37 @@ func (d *DockerExecutor) GetOutputStream(containerID string, combinedOutput bool
 	return reader, nil
 }
 
+func (d *DockerExecutor) GetExitCode(containerID string) (int32, error) {
```

Comment:
Why is the return type `int32` as opposed to `int`?

Reply:
I used int32 because I saw that K8S client method to get a container exit code returns an `int32`